ies-route53/recipes/add.rb

```diff
@@ -1,13 +1,15 @@
 instance = get_instance
 
+aws_config = get_aws_config
+
 ies_route53_record 'create a record' do
   name                  get_record_name
   value                 instance['ip']
   type                  'A'
   ttl                   node['ies-route53']['zone']['ttl']
-  zone_id               node['ies-route53']['zone']['id']
-  aws_access_key_id     node['ies-route53']['zone']['custom_access_key']
-  aws_secret_access_key node['ies-route53']['zone']['custom_secret_key']
+  zone_id               aws_config[:zone_id]
```

Comment:
Btw, this is the weirdest thing I have seen in a while.

Works: `aws_config[:zone]`
Doesn't work: `aws_config['zone']`

I wish I knew how Ruby worked.